Updated the list of relevant SNOW instances (#220)
hashicorp/terraform-docs-common: website/docs/cloud-docs/integrations/service-now/index.mdx

> This integration has been tested on the following ServiceNow server versions:

Reviewer: ```suggestion
You can use this integration on the following ServiceNow server versions:
```

Suggest changing "This integration has been tested" to "You can use this integration" to have an active voice.

Author: Good suggestion, thanks! Done.
